File: src/test/java/com/converter/api/model/ConversionTest.java
package com.converter.api.model;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import java.math.BigDecimal;

class ConversionTest {

    @Test
    void getDestinyValue() {
        Conversion conversion = new Conversion(new User(), "BRL", new BigDecimal("5"), "USD", new BigDecimal("3"));
        Assertions.assertEquals(new BigDecimal("15"), conversion.getDestinyValue());
    }
}

File: src/test/kotlin/com/converter/api/model/ConversionTest.kt
package com.converter.api.model

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import java.math.BigDecimal

internal class ConversionTest {

    @Test
    fun destinyValue() {
        val user = User(
            email = "a@a.com",
            password = "pass"
        )
        val conversion = Conversion(
            user = user,
            originCur = "BRL",
            originValue = BigDecimal("5"),
            destinyCur = "USD",
            conversionRate = BigDecimal("3")
        )
        Assertions.assertEquals(BigDecimal("15"), conversion.getDestinyValue())
    }
}